tests/SmartEntity/FlexEntityTest.php

```diff
@@ -36,7 +36,7 @@ public function testMake()
         $this->assertSame(1, $entity->getA());
     }
 
-    public function testNonExisitingMethod()
```

Comment:
Haha, nice catch. Was just trying to figure out what changed for a couple of seconds...

Reply:
;)